File: app/src/main/java/com/starostinvlad/fan/AppDatabase.java
package com.starostinvlad.fan;


import com.starostinvlad.fan.GsonModels.CurrentSerial;
import com.starostinvlad.fan.GsonModels.Datum;
import com.starostinvlad.fan.GsonModels.News;
import com.starostinvlad.fan.VideoScreen.PlayerModel.Episode;
import com.starostinvlad.fan.VideoScreen.PlayerModel.Season;
import com.starostinvlad.fan.VideoScreen.PlayerModel.Serial;
import com.starostinvlad.fan.VideoScreen.PlayerModel.Translation;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(
        entities = {
                Datum.class, News.class, CurrentSerial.class
        },
        version = 12,
        exportSchema = false
)
public abstract class AppDatabase extends RoomDatabase {
    public abstract DatumDao datumDao();

    public abstract CurrentSerialDao currentSerialDao();

    public abstract SearchedDao searchedDao();
}

File: app/src/main/java/com/starostinvlad/fan/AppDatabase.kt
package com.starostinvlad.fan

import androidx.room.Database
import androidx.room.RoomDatabase
import com.starostinvlad.fan.GsonModels.CurrentSerial
import com.starostinvlad.fan.GsonModels.Datum
import com.starostinvlad.fan.GsonModels.News
import kotlin.Throws

@Database(entities = [Datum::class, News::class, CurrentSerial::class], version = 12, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {
    abstract fun datumDao(): DatumDao?
    abstract fun currentSerialDao(): CurrentSerialDao
    abstract fun searchedDao(): SearchedDao
}